build/generate-manifests.sh

```diff
@@ -44,11 +44,15 @@ function checkDependencies() {
 }
 
 function generateCRDs() {
-  #go run -mod=vendor github.com/operator-framework/operator-sdk/cmd/operator-sdk generate crds --crd-version v1
   go run -mod=vendor sigs.k8s.io/controller-tools/cmd/controller-gen crd \
       paths=./pkg/apis/maistra/... \
       crd:maxDescLen=0,preserveUnknownFields=false,crdVersions=v1beta1 \
       output:crd:dir=./deploy/crds
+   # workaround for https://github.com/kubernetes-sigs/controller-tools/issues/457
+  #sed -i -e "s/\( *\)\(description\: The IP protocol for this port\)/\1default: TCP\n\1\2/" \
+  #    deploy/crds/maistra.io_servicemeshcontrolplanes.yaml
+  sed -i -e '/x-kubernetes-list-map-keys:/,/x-kubernetes-list-type: map/ { /- protocol/d }' deploy/crds/*
+  sed -i -e '/---/d' deploy/crds/*
```

Comment:
Maybe add a comment about why we're doing this.  It's not obvious, because it shouldn't be a problem applying the resource and there's nothing to tie it back to problems with OLM metadata builds.